frontend/src/components/Results.jsx

```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import { useLocation } from 'react-router';
 import axios from 'axios';
 import Post from './Post';
-
+import Comments from './Comments';
```

Comment:
Comments component is not functional right now. Please remove unfunctional components.